### PR TITLE
Add the ability to pad terms with whitespace, similar to printf

### PR DESCRIPTION
**Pull request by seddy**
A hacky, but allows you to use %5S or %-5S similar to printf/sprintr to right/left pad the various columns; useful specifically for severity, where INFO has four characters and DEBUG has 5 (meaning the actual logs all get misaligned).

Would be better to maybe split out each element into a class, but the whole file's a bit procedural so couldn't be bothered to make sweeping changes which would've been needed for that.

Happy to do that if anyone thinks it's worth the time?

### CODE REVIEW

**Comment by dougdroper** [comment]
+R, looks fine, must admit though that this gem is horrible written, could be interesting to fix it up, but not sure it is worth the time, unless you think there will be some benefit from doing so..

File: lib/grizzled/rails/logger.rb [hunk 1]
```diff
@@ -162,6 +168,20 @@ def do_add(severity, message, progname, options = {}, &block)
 
           add_without_grizzling(severity, message, progname, &block)
         end
+
+        def pad(check, string)
+          check_found = Configuration.format.match(/%(-?\d*)#{check}/)
```
Comment (ohthatjames) [inline lib/grizzled/rails/logger.rb]: Is there any way we can do this once, at initialization time?

Comment (seddy) [inline lib/grizzled/rails/logger.rb]: Not a bad idea!  Should be easy enough.

**Comment by seddy** [comment]
Note, pushed up another commit... but it's a initial commit as I've got some debugging to do (need to push up before I can test), so please ignore until I've fixed it.  Though I might leave this till later in the week now; spent enough time on it so far today :)!

**Comment by seddy** [comment]
Ha - just found this is a PR that's still open of mine. Don't care anymore and looks like there was a bug, closing.

**Pull request closed without merging**